web/src/hotkey.js

```diff
@@ -189,7 +189,9 @@ export function get_keydown_hotkey(e) {
         }
         return undefined;
     } else if (e.metaKey || e.ctrlKey) {
-        return undefined;
+        // If the pressed key is Ctrl+Enter, we want to send the message. Therefore, if the ASCII
+        // value of the key is 13 (Ctrl+Enter), we return the hotkey from keydown_cmd_or_ctrl_mappings.
+        return e.which === 13 && !e.shiftKey ? keydown_cmd_or_ctrl_mappings[e.which] : undefined;
```

Comment:
Can you explain the reasoning behind this change? I don't understand it at all; the case described in the comment should be where `isCmdOrCtrl && !e.shiftKey`, which is handled in the previous block.

Reply:
Earlier, we expected `Ctrl+Enter` and `Cmd+Enter` on Mac to do the same thing. However, after moving the key value 13 from the `keydown_ctrl_mappings` to `keydown_cmd_or_ctrl_mappings`, the above block that checks `isCmdOrCtrl` will return false in case the pressed button is `Ctrl+Enter`. As a result, this block will never run and the message won't be sent. To address this issue, the 194th line adds a condition to check if the pressed combination of keys has a value of 13 (`Ctrl+Enter`). If this condition is true, it will return the hotkey, which is what we expect it to do.

Reply:
I'm still not sure I understand; why is the bug not that we should be computing `isCmdOrCtrl` differently, or adjusting the check for the previous block in some other way?

Reply:
@timabbott , thanks for the review. I think we don't need to change the `isCmdOrCtrl` function since the only Ctrl hotkey combination that we need to go into the if block is `Ctrl+enter`. Therefore, in the if condition itself, we can simply check if the pressed combination has an ASCII value of 13 and if `ctrlKey` is true. If this condition is true, then we are sure that it is `Ctrl+enter`, and thus, we allow it to go into the if block. Please let me know if you have something else in mind.